File: scala-gatling-client/src/gatling/scala/io/swagger/client/model/FieldInstallationObject.scala

package io.swagger.client.model


case class FieldInstallationObject (
    _name: String,
    _fields: Option[List[FieldObject_fields]],
    _custom: Option[Boolean]
)
object FieldInstallationObject {
    def toStringBody(var_name: Object, var_fields: Object, var_custom: Object) =
        s"""
        | {
        | "name":$var_name,"fields":$var_fields,"custom":$var_custom
        | }
        """.stripMargin
}

File: dart-jaguar-client/lib/model/field_installation_object.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

import 'package:swagger/model/field_object_fields.dart';
part 'field_installation_object.jser.dart';

class FieldInstallationObject {
  
  final String name;
  
  final List<FieldObjectFields> fields;
  
  final bool custom;
  

  FieldInstallationObject(
    

{
    
     this.name = null,   this.fields = const [],  
     this.custom = null 
    
    }
  );

  @override
  String toString() {
    return 'FieldInstallationObject[name=$name, fields=$fields, custom=$custom, ]';
  }
}

@GenSerializer()
class FieldInstallationObjectSerializer extends Serializer<FieldInstallationObject> with _$FieldInstallationObjectSerializer {

}
